src/server/pingserver-rs/tests/integration.rs

```diff
@@ -26,7 +32,7 @@ fn main() {
 
     // wait for server to startup. duration is chosen to be longer than we'd
     // expect startup to take in a slow ci environment.
-    std::thread::sleep(Duration::from_secs(10));
+    std::thread::sleep(STARTUP_DELAY);
```

Comment:
I wonder if we should eventually move to polling a "ready-to-serve" flag/variable.

Reply:
Yeah - this delay definitely feels long during development. Will think about how to achieve this - we'll likely have common testing functionality as more servers are implemented.